From a701ca72d96829d300405117b31f73dccc40dc92 Mon Sep 17 00:00:00 2001
From: Michael Heimpold <mhei@heimpold.de>
Date: Tue, 27 Feb 2024 07:48:34 +0100
Subject: [PATCH] Add missing find_package for date

Signed-off-by: Michael Heimpold <mhei@heimpold.de>
---
 CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 64d575359d40..6c5e851afc1e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -33,6 +33,7 @@ if (NOT DISABLE_EDM)
     # In EDM mode, we can't install exports (because the dependencies usually do not install their exports)
     set(EVSE_SECURITY_INSTALL OFF)
 else()
+    find_package(date REQUIRED)
     find_package(everest-log REQUIRED)
     find_package(everest-timer REQUIRED)
 endif()
-- 
2.34.1

