From 021e4f066a06fb4cd9ed27a6a8c581c49fee9e53 Mon Sep 17 00:00:00 2001
From: Kai-Uwe Hermann <kai-uwe.hermann@pionix.de>
Date: Wed, 21 Sep 2022 18:31:07 +0200
Subject: [PATCH] disable everestjs

Signed-off-by: Kai-Uwe Hermann <kai-uwe.hermann@pionix.de>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index d8c134e..5296125 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -70,7 +70,7 @@ add_subdirectory(src)
 add_subdirectory(schemas)
 
 # everest javascript wrapper
-add_subdirectory(everestjs)
+# add_subdirectory(everestjs)
 
 # FIXME (aw): should this be installed or not?  Right now it is needed for the
 #             current packaging approach
-- 
2.37.3

