From e045fabe302662517e295d7e4c1f5700b6e7a6a9 Mon Sep 17 00:00:00 2001
From: Kai-Uwe Hermann <kai-uwe.hermann@pionix.de>
Date: Thu, 22 Sep 2022 14:08:54 +0200
Subject: [PATCH 2/2] disable auth tests

Signed-off-by: Kai-Uwe Hermann <kai-uwe.hermann@pionix.de>
---
 modules/Auth/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/Auth/CMakeLists.txt b/modules/Auth/CMakeLists.txt
index 8d6e5b9..92008e1 100644
--- a/modules/Auth/CMakeLists.txt
+++ b/modules/Auth/CMakeLists.txt
@@ -32,5 +32,5 @@ target_sources(${MODULE_NAME}
 
 # ev@c55432ab-152c-45a9-9d2e-7281d50c69c3:v1
 # insert other things like install cmds etc here
-add_subdirectory(tests)
+# add_subdirectory(tests)
 # ev@c55432ab-152c-45a9-9d2e-7281d50c69c3:v1
-- 
2.37.3

