From 8a3d9f91fb0ed880e86f11614388895c664e15d9 Mon Sep 17 00:00:00 2001
From: Kai-Uwe Hermann <kai-uwe.hermann@pionix.de>
Date: Thu, 22 Sep 2022 17:43:56 +0200
Subject: [PATCH 7/7] c++ 17

Signed-off-by: Kai-Uwe Hermann <kai-uwe.hermann@pionix.de>
---
 modules/SerialCommHub/CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/modules/SerialCommHub/CMakeLists.txt b/modules/SerialCommHub/CMakeLists.txt
index 4144525..c12c558 100644
--- a/modules/SerialCommHub/CMakeLists.txt
+++ b/modules/SerialCommHub/CMakeLists.txt
@@ -15,6 +15,8 @@ target_link_libraries(${MODULE_NAME}
     everest::modbus
     date::date
 )
+
+target_compile_features(${MODULE_NAME} PUBLIC cxx_std_17)
 # ev@bcc62523-e22b-41d7-ba2f-825b493a3c97:v1
 
 target_sources(${MODULE_NAME}
-- 
2.37.3

